.github/workflows/dogfood-deploy.yml

```diff
@@ -3,8 +3,8 @@ name: Deploy Dogfood Environment
 on:
   workflow_dispatch:
     inputs:
-      IMAGE_TAG:
-        description: 'The image tag wished to be deployed.'
+      DOCKER_IMAGE:
+        description: 'The full name of the docker to be deployed. (e.g. fleetdm/fleet:v4.24.1)'
```

Comment:
```suggestion
        description: 'The full name of the docker image to be deployed. (e.g. fleetdm/fleet:v4.24.1)'
```

Reply:
Also, looks like elsewhere this defaults to `fleetdm/fleet:main` should we just set this as a default instead of using an example?

```
        default: 'fleetdm/fleet:main'
```